packages/plugin-core/src/common/EngineNoteProvider.ts

```diff
@@ -19,7 +19,7 @@ import {
   TreeItemCollapsibleState,
```

Comment:
Can you move all the files that are a part of this review from this location to `src\views\common\treeview` ?